rnd/autogpt_builder/.env.example

```diff
@@ -1 +1,7 @@
 AGPT_SERVER_URL=http://localhost:8000/api
+
+## Supabase credentials
+## YOU ONLY NEED THEM IF YOU WANT TO USE SUPABASE USER AUTHENTICATION
+## If you're using self-hosted version then you most likely don't need to set this
+# NEXT_PUBLIC_SUPABASE_URL=your-project-url
+# NEXT_PUBLIC_SUPABASE_ANON_KEY=your-anon-key
```

Comment:
is it possible to default these to our instance safely

Reply:
I don't think so